### DISCUSSION TITLE
Show only added and removed lines

### DISCUSSION THREAD

**Discussion by dageci** [WinMerge/winmerge#1937] [Q&A]
Hello,
first I love WinMerge and thanks to all the people that are contibuting for making this tool so helpfull.
I'm comparing 2 xml files that are translation files.
Is there a way to only show the lines that were added or removed, because they all have the same xml tags but with different values?

**Comment by sdottaka** [comment]
It may be a little different from the functionality you expect, but you can select View→Diff ontext→0 Lines menu item to show only the lines with differences.

![image](https://github.com/WinMerge/winmerge/assets/98126/3db1da70-5ff1-439b-a724-c8a128f30a0a)
